test/e2e/alcotest/passing/verbose_newlines.ml

```diff
@@ -0,0 +1,29 @@
+(** Reproduction for https://github.com/mirage/alcotest/issues/225, testing the
+    interaction between `--verbose` and newlines in the test stdout. *)
+
+let () =
+  Alcotest.run ~verbose:true "suite-name"
```

Comment:
Is `~verbose` exactly the same as passing `--verbose` on the CLI?

Reply:
It's not tested, but it should be. I can alter the test to work directly with `--verbose` instead.

Reply:
With https://github.com/mirage/alcotest/pull/229, the two are tested to be the same.